### PR TITLE
Feat: Data Export Test

### PR DESCRIPTION
**Pull request by haanh764**
What's changed:
- `backend/data_export/tests/test_catalog.py`: added `ArticleAnnotation` project to test catalog
- `backend/data_export/tests/test_task.py`: added new test tasks for data export of ArticleAnnotation Project and Custom Relation Extraction

### CODE REVIEW

File: backend/data_export/tests/test_task.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from cmath import nan
 import os
```
Comment (teddy-f-47) [inline backend/data_export/tests/test_task.py]: I think the `nan` is accidentally imported and can be removed XD

**Review by teddy-f-47 — APPROVED** [review]
Thank you for the great work! I think the code looks fine, and I have tested using the new tests and everything is fine. There is just one minor comment about removing an import. XD Nevertheless, I give my approval for merging now. :D

File: backend/data_export/tests/test_task.py (hunk 1 above)
Comment (haanh764) [reply backend/data_export/tests/test_task.py]: yeah it's accidentally imported XD I've removed it

**Review by artikandri — APPROVED** [review]
I'm sorry, I have read this but I forgot to give my approval :"   Approved for merging, great work as always. Thank you!

**Pull request merged**